.github/workflows/gomu-gomu-tests.yml

```diff
@@ -52,3 +53,52 @@ jobs:
           cd ../gomu-gomu-no-gatling
           RUST_LOG=info cargo run -- shoot -c config/default.yaml
           kill $MADARA_RUN_PID
+
+      - name: Process results
+        run: |
+          set -v -x +e
+          cp ../gomu-gomu-no-gatling/report.json ./
+          #### Extract TPS, UOPS, Extrinsics and Steps from the report ####
+          jq ' 
+          .benches[] |= (.name as $benchName | .metrics[] |= (.name = .name + " ("+ $benchName + ")")) # Append benchmark name to each metric name
+          | .benches[].metrics[].extra = .extra # set extra field for each metric
+          |.benches # extract only benches
+          |map(.metrics)|add # merge metrics from different benches
+          |map(select(.name | test("Average TPS|Average UOPS|Average Extrinsics|Average Steps"))) #filter only the metrics we want
+          ' report.json > processed_report.json
+
+      - name: Check for failed requests
+        run: |
+          set -v -x +e
+          #### Extract Failed from the report ####
+          jq ' 
+          .benches[] |= (.name as $benchName | .metrics[] |= (.name = .name + " ("+ $benchName + ")")) # Append benchmark name to each metric name
+          | .benches[].metrics[].extra = .extra # set extra field for each metric
+          |.benches # extract only benches
+          |map(.metrics)|add # merge metrics from different benches
+          |map(select(.name | test("Failed"))) # filter only the metrics we want
+          |any(.[]; .value > 0) | if . then halt_error(1) else {} end # fail if there are failed requests
+          ' report.json
```

Comment:
I have no idea what I'm reading.

Is it some standard thing you found online, or do you write this yourself all custom?

Reply:
Unfortunately I had to write it custom, since the benchmark processing action doesn't support our output format. I've commented it extensively in the hopes of making it a bit more readable.

Reply:
Could we add a gh-compatible output format to gomu-gomu?